DigitalLearningSolutions.Web/Controllers/HomeController.cs

```diff
@@ -63,24 +64,30 @@ public IActionResult LearningContent()
         {
             var publicBrands = brandsService.GetPublicBrands()
                 .Select(b => new LearningContentSummary(b));
-
             var model = new LearningContentLandingPageViewModel
             {
                 MiniHubNavigationModel = new MiniHubNavigationModel(LandingPageMiniHubName, sections, 2),
                 UserIsLoggedIn = User.Identity.IsAuthenticated,
+                UserIsLoggedInCentre = GetCentreId() != null,
                 CurrentSiteBaseUrl = configuration.GetCurrentSystemBaseUrl(),
                 LearningContentItems = publicBrands,
             };
 
             return View(model);
         }
 
+        private int? GetCentreId()

```

Comment:
The `Web\Helpers\CustomClaimHelper.cs` class has a method that returns nullable centre ID of logged in user directly (GetCentreId) we should use this rather than than duplicating the code of that method here.